flowzone.yml

```diff
@@ -41,7 +54,6 @@
     with:
       fetch-depth: 0
       submodules: "recursive"
-      token: ${{ secrets.FLOWZONE_TOKEN }}
```

Comment:
Why were these removed? If the intent is to use `${{ secrets.GITHUB_TOKEN }}` instead I would rather be clear about that rather than appear to have forgotten to specify.

Reply:
That was the intention.

Reply:
In my other PR I removed all permissions from this token at the workflow level. So this will break on private repos since we removed read access.

Fortunately you can just set new permissions at the versioned_source and tagged_source jobs to get what you need. Or change the default at the top of the workflow and keep docker_test with empty permissions.